src/routes/clients.js

```diff
@@ -36,23 +37,31 @@ router.post('/', (req, res) => {
                     JSON.stringify(errors));
   }
 
-  clients.create(req.body.name).then(client => {
-    res.status(201).send(client);
-  }).catch(error => {
-    if (error.message && error.message === RECORD_ALREADY_EXISTS) {
-      return ApiError(res, 403, ERRNO_FORBIDDEN, FORBIDDEN);
-    }
-    ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR);
+  db().then(models => {
+    models.Clients.create(req.body).then(client => {
+      res.status(201).send(client);
+    }).catch(error => {
+      if (error.name && error.name === modelErrors[RECORD_ALREADY_EXISTS]) {
+        return ApiError(res, 403, ERRNO_FORBIDDEN, FORBIDDEN);
+      }
+      ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR,
+               error.message + ' ' + JSON.stringify(error.errors));
+    });
   });
 });
 
 // Get the list of registered API clients.
 router.get('/', (req, res) => {
-  clients.getAll().then(clients => {
-    res.status(200).send(clients);
-  }).catch(error => {
-    ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR);
-  })
+  db().then(models => {
+    models.Clients.findAll({
+      attributes: ['key', 'name']
+    }).then(clients => {
+      res.status(200).send(clients);
+    }).catch(error => {
+      ApiError(res, 500, ERRNO_INTERNAL_ERROR, INTERNAL_ERROR,
```

Comment:
Ah, yes. That's a good point. I don't think sequelize will dump any private content in this case, but I guess it's better to play safe in this case.